libs/ledger-live-common/.unimportedrc.json

```diff
@@ -303,7 +303,8 @@
     "src/index.ts",
     "src/jest.d.ts",
     "src/postOnboarding/mock.ts",
-    "src/reactNativeSvg.d.ts"
```

Comment:
@valpinkman we need conventions on our file structure otherwise it will be hard to maintain. Something like __tests__/mock.ts in that case we will be able to exclude __tests__ pattern.

Reply:
we could also consider currencies/mock to be a valid entry point btw, because it is somehow (for the tests).
the problem rely more on the fact live-common is a multi entry point library.